org.frescobaldi.Frescobaldi.yaml

```diff
@@ -0,0 +1,30 @@
+---
+app-id: org.frescobaldi.Frescobaldi
+
+base: io.qt.qtwebengine.BaseApp
+base-version: '5.14'
+
+runtime: org.kde.Platform
+runtime-version: '5.14'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.texlive  # version 19.08
+
+command: frescobaldi
+
+finish-args:
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin
+  - --filesystem=home
+  - --share=ipc
+  - --share=network
+  - --device=all
+  - --socket=pulseaudio
+  - --socket=x11
+# See Wayland bugs:
+# https://github.com/frescobaldi/frescobaldi/issues?q=is%3Aissue+is%3Aopen+wayland+in%3Atitle
+#  - --socket=wayland
+#  - --env=QT_QPA_PLATFORM=xcb
+
+modules:
```

Comment:
Not a fan of splitting the modules into multiple files

Reply:
I thought it would have been cleaner.. and also easier to copy for someone who may happen to build some flatpak app which requires lilypond. But I can merge the files, no problem.

Reply:
Done